services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/design-kits.js

```diff
@@ -47,13 +71,76 @@ const DesignKits = ({ libraryData, navData }) => {
     )
   }
 
+  const noDesignKits = (
+    <>
+      <PageDescription>This library contains no compatible design kits.</PageDescription>
+      <P>
+        It looks like there are no design kits compatible with this library. Explore the design kit
+        catalog to find more design kits or contact libray maintainers.
+      </P>
+      <CardGroup>
+        <Column lg={4} md={4}>
+          <ResourceCard subTitle="Design kit catalog" href="/design-kits">
+            <RulerAlt size={32} />
+          </ResourceCard>
+        </Column>
+      </CardGroup>
+    </>
+  )
+
   return (
     <>
       <NextSeo {...seo} />
       <PageHeader bgColor={pageHeader?.bgColor} title="Design kits" pictogram={pageHeader?.icon} />
       <Grid>
         <Column sm={4} md={8} lg={12}>
-          <div className={pageStyles.content}>Coming soon!</div>
+          <div className={pageStyles.content}>
+            {!designKits && noDesignKits}
+            {designKits && (
+              <>
+                <PageDescription>
+                  The following design kits are compatible with the {name} library. If you are new
+                  to Carbon, check out the{' '}
+                  <Link href="/designing/get-started" passHref>
+                    <CarbonLink>design guidance</CarbonLink>
+                  </Link>{' '}
+                  to get started. You can also view all design kits in the{' '}
+                  <Link href="/design-kits" passHref>
+                    <CarbonLink>catalog</CarbonLink>
+                  </Link>
+                  .
+                </PageDescription>
+                <AnchorLinks>
+                  {filteredDesignTools.length > 2 &&
+                    filteredDesignTools.map((item, i) => (
+                      <AnchorLink key={i}>{get(designTools, `[${item}].name`)}</AnchorLink>
+                    ))}
+                </AnchorLinks>
+                {filteredDesignTools.map((item, i) => (
+                  <>
+                    <H2 key={i}>{get(designTools, `[${item}].name`)}</H2>
+                    <CardGroup>
+                      {groupedDesignKits[`${item}`].map((kit, i) => (
+                        <Column lg={4} md={4} key={i}>
```

Comment:
Small breakpoint doesn't seem to follow redline 
<img width="571" alt="image" src="https://user-images.githubusercontent.com/40550942/189402046-b69c1e9e-a7b4-479e-8bd6-d43c46c9a41c.png">

<img width="488" alt="image" src="https://user-images.githubusercontent.com/40550942/189401316-1a22d9f7-d15f-4b0f-992a-ba92027eaa5b.png">

Reply:
oops! was missing sm={4} fixed